deploy/local/dal

```diff
@@ -26,79 +26,48 @@ if { $tcl_version < $min_tcl_ver } {
+
+module-whatis "Name: Intel® oneAPI Data Analytics Library"
+module-whatis "Version: $modulefilename/$modulefilever"
+module-whatis "Description: Library for building compute-intense applications optimized for Intel CPUs and GPUs."
+module-whatis "URL: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal.html"
+module-whatis "Dependencies: none"
+
+proc ModulesHelp { } {
+    global modulefilename
+    global modulefilever
+    module whatis "${modulefilename}/${modulefilever}"
 }
 
-# if fullpath contains links, resolve them
-set scriptpath "[file normalize "$scriptpath"]"
-
-# get directory holding this modulefile script and others
-set modulefileroot "[file dirname "$scriptpath"]"
-
-# get name of modulefile script we are loading
-set modulefilename "[file tail "$scriptpath"]"
-
-# determine modulefile script version
-set modulefilever "[file dirname "$modulefileroot"]"
-set modulefilever "[file tail "$modulefilever"]"
-
-# point to component top-level root folder
-set componentroot "[file dirname "$modulefileroot"]"
-set componentroot "[file dirname "$componentroot"]"
-
-# get component folder name
-set componentname "[file tail "$componentroot"]"
-
-# get oneAPI top-level root folder
-set oneapiroot "[file dirname "$componentroot"]"
-
-# disallow loading multiple versions of this modulefile
-# disallow loading multiple architectures of this modulefile
-# if only 64-bit architecture exists the test still works
-set mname32 $modulefilename
-set mname64 [string trimright $mname32 "32"]
-if { [string equal "$mname32" "$mname64"] } {
-    append mname32 "32"
-}
-conflict $mname32
-conflict $mname64
+##############################################################################
 
-
-# On load print component name and version being loaded
-if { [ module-info mode load ] } {
-    puts stderr "Loading $modulefilename version $modulefilever"
-}
-
-# On `module unload` print component module name and version being removed
-# Include `module list` message only if this modulefile loads dependent modules
-if { [ module-info mode ] == "unload" || [ module-info mode ] == "remove" } {
-    puts stderr "Removing $modulefilename version $modulefilever"
-    puts stderr "Use `module list` to view any remaining dependent modules."
-}
-
-
-# ###### Component Specific env vars setup ###################################
+# Define environment variables needed for an isolated component install.
 
 # Set intermediate variables
-set daalroot "$componentroot/$modulefilever"
-set daal_target_arch         "intel64"
-
-module-whatis "Intel(R) oneAPI Data Analytics Library for $daal_target_arch."
+set dalroot "$componentroot"
+set daal_target_arch "intel64"
 
 # Setup environment variables
-setenv          DAL_MAJOR_BINARY   __DAL_MAJOR_BINARY__
-setenv          DAL_MINOR_BINARY   __DAL_MINOR_BINARY__
-setenv          DALROOT            $daalroot
-setenv          DAALROOT           $daalroot
```

Comment:
We still have DAALROOT used in scikit, so we have to make changes there as well

https://github.com/search?q=repo%3Aintel%2Fscikit-learn-intelex%20DAALROOT&type=code

+ Probably check other places like automation, installer validation